int32 grassNum
float32 x
float32 y
int32 growth
